pkg/storage/redis.go

```diff
@@ -62,9 +68,73 @@ func (b *RedisDB) Close() error {
 	return b.db.Close()
 }
 
+type TxContext struct {
+	tx        goredislib.Pipeliner
+	watchKeys []string
+	watchOnly bool
```

Comment:
Perhaps this is unused? I don't see where it's being set to true.

Reply:
Removed